src/Test/BattleshipModelTest.java

```diff
@@ -212,6 +212,35 @@ public void smartShootAtPlayer(){
 
     }
 
+    @Test
+    public void easyComputerFire(){
```

Comment:
This test is really clean. The code you've produced here is probably the best I've seen so far on our project. I appreciate it.